CuratedContent/SoftwareSustainabilityInstituteGuides.md

```diff
@@ -0,0 +1,32 @@
+# Software Sustainability Institute Guides 
+<!-- deck text start -->
+ 
+The Software Sustainability Institute presents over 80 guides that provide
+advice for software development processes and techniques for each research software role so that your team can ensure software sustainability.
+
+<!-- deck text end --> 
+
+#### Contributed by [Patricia Grubel](https://github.com/pagrubel "Patricia Grubel")
+#### Publication date: September 24, 2021
+
+Resource information | Details
+:--- | :--- 
+Website | [Software Sustainability Institute Guides](https://www.software.ac.uk/resources/guides)
+
+The Software Sustainability Institute (SSI), based in the UK,
+promotes the importance of scientific research software. The SSI
+website presents over 80 guides to aid your team to protect
+software into the future. The guides are categorized by the
+following research software project roles: researchers,
+developers, managers and instructors or material developers. The
+categories and descriptions make it easy to navigate to guides
+that are valuable to each of your team members. 
+
+Managers, researchers and developers can peruse a variety of topics such as building community, recruiting champions and students, managing research projects, data management plans and approaches to sustainability. Developers can take a deeper dive on subjects like making software reusable, publicizing software, best practices, and the challenges of dependencies. While researchers will find topics on publishing and the importance of openness in research helpful. The site also hosts guides for training and training materials including specific help for organizing and teaching Carpentry workshops. 
```

Comment:
> While researchers will find topics on publishing and the importance of openness in research helpful.

@pagrubel, I don't think that is a complete sentence.  I think you can make this a sentence by removing the word "While".

Reply:
@pagrubel, the fix is in the [stacked PR](https://bssw.io/items/pull-request-size-matters#sfer-ezikiw-7) #1075.  That PR just needs to be accepted and merged to this PR branch and then this PR can be approved and merged.